docs/docs/frigate/installation.md

```diff
@@ -237,3 +237,41 @@ It is recommended to run Frigate in LXC for maximum performance. See [this discu
 ## ESX
 
 For details on running Frigate under ESX, see details [here](https://github.com/blakeblackshear/frigate/issues/305).
+
+## Synology NAS on DSM 7
+you can run Frigate NVR on DSM 7.x. tested on DSM 7.1.1-42962 Update 4
+
+
+**General:**
+
+you need to tick the "Execute container using high privilage" checkbox in order to give the frigate container the elevated privilages it may need.
+if you want the container to automatically restart whenever it improperly shuts down then you can tick the "Enable auto-restart" checkbox.
```

Comment:
```suggestion

The `Execute container using high privilage` option needs to be enabled in order to give the frigate container the elevated privileges it may need.

The `Enable auto-restart` option can be enabled if you want the container to automatically restart whenever it improperly shuts down due to an error.	
```